sympy/geometry/line.py

```diff
@@ -2365,7 +2366,10 @@ class Segment2D(LinearEntity2D, Segment):
     Point2D(5/2, 2)
 
     """
-    def __new__(cls, p1, p2, **kwargs):
+    
```

Comment:
[The failing test](https://travis-ci.org/github/sympy/sympy/jobs/745076406#L325) is saying this training whitespace should be deleted.